cmd/display.go

```diff
@@ -77,5 +81,6 @@ func init() {
 	// Cobra supports local flags which will only run when this command
 	// is called directly, e.g.:
 	// transformCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
-	displayCmd.Flags().StringVarP(&displayType, "type", "t", "charts", "charts or table")
+	displayCmd.Flags().StringVarP(&displayType, "mode", "m", TYPE_CHART, fmt.Sprintf("display mode [%s|%s]", TYPE_CHART, TYPE_TABLE))
+	displayCmd.Flags().StringVarP(&chartType, "type", "t", "", "chart type [line] (default line)")
```

Comment:
let's make `line` to be a default type explicitly here, via:
```
displayCmd.Flags().StringVarP(&chartType, "type", "t", "line", "chart type [line]")
```

Reply:
that would also add `(default line)`  help string automatically
not sure if we want to bother checking for the value to be one of those we support

Reply:
pls ignore as such approach would interfere with `--mode table`